src/conditional-flow/numeric-conditions.js

```diff
@@ -4,21 +4,21 @@
 function isArrayEmpty (array) {
 
   // TODO: write code in this function body to pass the tests
+  return array.length === 0;
 
 }
 
 // This function should return true if num1 is greater than num2, false otherwise
 function isGreaterThan (num1, num2) {
 
   // TODO: write code in this function body to pass the tests
-
+  return num1 > num2;
 }
 
 // This function should return the lowest number in the passed array
 function findLowest (nums) {
-
   // TODO: write code in this function body to pass the tests
-
+  return Math.min(...nums)
```

Comment:
This works :)
May fail for very big arrays. Might be worth adding an implementation that uses a loop?